tc/core/polyhedral/memory_promotion.cc

```diff
@@ -79,15 +119,21 @@ std::unique_ptr<TensorReferenceGroup> TensorReferenceGroup::makeSingleton(
 
 isl::set TensorReferenceGroup::approximateFootprint() const {
   auto scopedDomain = scopedAccesses().domain();
-  auto space = box.get_space();
+  auto space = approximation.box.get_space();
```

Comment:
Huh?
This looks like it belongs to the previous commit....
...indeed, 5d48a371948b9bfb0c14859db93d2569fc327547 doesn't compile.